intercom-java/src/main/java/io/intercom/api/Conversation.java

```diff
@@ -196,6 +196,9 @@ private static boolean isAdminQuery(Map<String, String> params) {
     @JsonProperty("updated_at")
     private long updatedAt;
 
+    @JsonProperty("waiting_since")
```

Comment:
do we want to start adding `ignore property` =?

Reply:
`Conversation` already has `@JsonIgnoreProperties(ignoreUnknown = true)`
I'll add it to `Notification` in another PR